src/Backend/Modules/Pages/Actions/Add.php

```diff
@@ -550,12 +554,33 @@ private function validateForm(): void
         }
     }
 
-    private function getImage(bool $allowImage): ?string
+    private function getImage(bool $allowImage, ?string $defaultImage): ?string
```

Comment:
wouldn't it make more sense to use `string $defaultImage = null`

Reply:
Data comes straight of out of database, and will always be there, so no need to add default value I think.